knitr/reduce-sum/reduce_sum_tutorial.Rmd

```diff
@@ -267,36 +267,29 @@ logistic1 <- cmdstan_model("logistic1.stan", cpp_options = list(stan_threads = T
 
 **Note:** If you get an error `Error in self$compile(...) : unused argument (cpp_options = list(stan_threads = TRUE))`, [update](https://mc-stan.org/cmdstanr/) your `cmdstanr` to the latest version (the threading interface was changed in May 2020, shortly after this tutorial was originally published).
 
-
-Set the number of threads each chain will use with `set_num_threads`:
-
-```{r}
-set_num_threads(2)
-```
-
-As this computer has 8 cores and we intend to run the usual 4 chains,
-we will use 2 threads per chain to make full use of the processor
-(4 chains with 2 threads each can make use of the full eight cores).
-The within-chain parallelism is generally less effiecient as compared to
-running chains in parallel, but if greater single-chain speedups are desired,
-then the user can choose to run fewer chains (i.e. 2 chains with 4 threads
-each) or use more machines.
+The computer I'm on has 8 cores and we want to make use of all of them. Running 4
+chains in parallel gives us 2 threads per chain (which makes full
+use of all 8 cores on the processor). The within-chain parallelism is generally
+less efficient as compared to running chains in parallel, but if greater
+single-chain speedups are desired, then the user can choose to run fewer
+chains (i.e. 2 chains in parallel with 4 threads each).
 
 Run and time the model with:
 
 ```{r}
 redcard_data$grainsize <- 1
 time1 = system.time(fit1 <- logistic1$sample(redcard_data,
-                                             cores = 4,
                                              chains = 4,
+                                             parallel_chains = 4,
+                                             threads_per_chain = 2,
                                              refresh = 1000))
 
 time1
 ```
-**Note:** Older versions of cmdstanr use `num_cores` and `num_chains` instead of `cores` and `chains`. If you get an error, [update](https://mc-stan.org/cmdstanr/) `cmdstanr`.
+**Note:** Older versions of cmdstanr use `num_cores`, `cores`, and `num_chains` instead of `parallel_chains`, `threads_per_chain`, and `chains`. If you get an error, [update](https://mc-stan.org/cmdstanr/) `cmdstanr`.
```

Comment:
Not sure we need this note every time, probably the first time is enough (you already have it above), but I'm fine either way.

Reply:
Fair point. Keep it though.

Reply:
Ok let's keep it. I'll merge these now.